File: grading-students/grading-students.py
#https://www.hackerrank.com/challenges/three-month-preparation-kit-grading/problem?isFullScreen=true&h_l=interview&playlist_slugs%5B%5D=preparation-kits&playlist_slugs%5B%5D=three-month-preparation-kit&playlist_slugs%5B%5D=three-month-week-two

def gradingStudents(grades):
    # Write your code here
    rounded_grade = []
    for grade in grades:
        if grade < 38 or grade % 5 < 3:
            rounded_grade.append(grade)
        else:
            rounded_grade.append(grade + (5 - grade % 5))
    return rounded_grade

File: grading-students/grading-students.js
function gradeRounding(grade) {
    if (grade < 38) {
        return grade
    }
    if (grade % 5 < 3) {
        return grade
    }
    const result = grade + (5 - grade % 5);
    return result
}


function gradingStudents(grades) {
    // Write your code here
    return grades.map((grade) => gradeRounding(grade))
}